python/ray/autoscaler/_private/resource_demand_scheduler.py

```diff
@@ -822,18 +814,19 @@ def get_nodes_for(
 
         # Give up, no feasible node.
         if not utilization_scores:
-            # TODO (Alex): We will hit this case every time a placement group
-            # starts up because placement groups are scheduled via custom
-            # resources. This will behave properly with the current utilization
-            # score heuristic, but it's a little dangerous and misleading.
-            logger.warning(
-                f"The autoscaler could not find a node type to satisfy the "
-                f"request: {resources}. If this request is related to "
-                f"placement groups the resource request will resolve itself, "
-                f"otherwise please specify a node type with the necessary "
-                f"resource "
-                f"https://docs.ray.io/en/master/cluster/autoscaling.html#multiple-node-type-autoscaling."  # noqa: E501
-            )
+            if not any(
+                is_placement_group_resource(resource)
+                for resources_dict in resources
+                for resource in resources_dict
+            ):
+                logger.warning(
+                    f"The autoscaler could not find a node type to satisfy the "
+                    f"request: {resources}. If this request is related to "
+                    f"placement groups the resource request will resolve itself, "
```

Comment:
Can we remove the placement group wording now?

Reply:
That's my only complaint, otherwise lgtm.

Reply:
good point. also removed the link because it seems to be dead now (and I can't find where it moved to..."